static/app/components/avatarChooser.tsx

```diff
@@ -147,13 +191,12 @@ class AvatarChooser extends React.Component<Props, State> {
     if (allowGravatar) {
       choices.push(['gravatar', t('Use Gravatar')]);
     }
-
     return (
       <Panel>
-        <PanelHeader>{t('Avatar')}</PanelHeader>
+        <PanelHeader>{title ?? t('Avatar')}</PanelHeader>
```

Comment:
Using `??` returns an empty string when `title` is an empty string. 
```suggestion
        <PanelHeader>{title || t('Avatar')}</PanelHeader>
 ```

Reply:
also might want to return `tct('[title]', {title})`

Reply:
Thanks! fixed